securedrop/sass/_base.sass

```diff
@@ -89,69 +95,9 @@
   +clearfix
   +center-text-align
   +button-centering
-
-  aside, .flash, .banner-warning
-    border-radius: 10px
-
-    i
-      font-size: 18pt
-
-  aside
-    background-color: #fffbe6
-    color: #82572d
-
-  aside, .flash
-    padding: 10px
-    margin: 10px
-    text-align: left
-    font-size: medium
-
-    &:dir(rtl)
-      text-align: right
-
-  .flash
-    display: flex
-    flex-flow: row nowrap
-    align-items: flex-start
-
-    &.success
-      border: 1px solid #E1F1E5
-      background-color: #E6FFEB
-
-      strong
-        color: #299549
-
-    &.notification
-      border: 1px solid #8ed9f6
-      background-color: #e3f7fc
-      font-size: medium
-      font-weight: normal
-
-      i
-        color: #4f7685
-
-    &.error
-      border: 1px solid $color_error_border
-      background-color: $color_error_background
-
-      i
-        color: #D62727
-
-    &.important
-      border: 1px solid #EBDCEB
-      background: #FDFAFD
-
-      strong
-        color: #673466
-      p
-        color: #555555
-
-  .banner-warning
-    border: 1px solid $color_error_border
-    background-color: $color_error_background
-
-    i
-      color: #D62727
+  +aside
+  +flash
+  +banner-warning
```

Comment:
:+1: the refactored aside, flash and banner-warning were carefully verified to exactly contain the concatenation of the corresponding combined selectors. While combined selectors are a terse way to express the same (i.e. more DRY), the motivation behind this refactor is most probably maintainability. When someone fixes or improves **flash** it is easier for them to not be forced to think about how it will influence **banner-warning** or **aside**. There is no reason why those would always share part of their CSS.